### DISCUSSION TITLE
bash: ganache-cli: command not found on Debian

### DISCUSSION THREAD

**Discussion by programonauta** [smartcontractkit/full-blockchain-solidity-course-py#571] [Q&A] [answered]
Hi I'm using Debian 11.

I've installed yarn successfully, and it looks like the ganache-cli was installed ok as well.

But when I try to run it I got this error:

```
ricardo@debian:~ ()$ yarn global add  ganache-cli
yarn global v1.22.17
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Installed "ganache-cli@6.12.2" with binaries:
      - ganache-cli
Done in 10.21s.
ricardo@debian:~ ()$ ganache-cli --version
bash: ganache-cli: command not found
```

The yarn was installed on `usr/local/bin`:

```
ricardo@debian:~ ()$ which yarn
/usr/local/bin/yarn
```

I found out the `ganache-cli` binary on `~/.config/yarn/global/node_modules/.bin` directory.

Does It work if I just put this directory on `PATH` env variable? 

Is it a workaround or there is some other way to do that?

**Comment by cromewar** [comment, marked as answer]
Hello @programonauta, yes the reason is the yarn node modules aren't added to your path so they wont work, this should work though:

`export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"`

Best regards, Cromewar.

**Reply by programonauta** [reply]
Thanks @cromewar
